### ISSUE TITLE
Can we write partition columns after data columns when merging the schema?

### ISSUE THREAD

**Issue by xianyinxin** [delta-io/delta#508]
Suppose we have schema (name string, id int) and (id) is the partition column. We write a new dataframe which has a new column, say (dt string), after merging the schema, the schema string in metastore looks like (name string, id int, dt string). Can we write it to (name string, dt string, id int)?

**Comment by liwensun** [comment]
Yes - I don't think you need to worry about column ordering when writing to Delta. Try it and find out. :)
